### PR TITLE
remove attendance policy

### PR DESCRIPTION
**Pull request by sheraaronhurt**
we do not collect attendance

### CODE REVIEW

**Comment by Denubis** [comment]
It may be worth noting that many local hosts *do* track attendance, either for compliance or local stats or institutional goals -- and that this is a specific matter to liaise with the host about? (And it's really fun when the physical sign-in sheets get lost...)

**Comment by brownsarahm** [comment]
Helping the host collect attendance info is listed in the [instructor checklist in the handbook](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html#instructor-checklist) and that is a part of this episode later on. Is that enough emphasis on that @Denubis?

File: _episodes/21-carpentries.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ You may only call a workshop a Software Carpentry, Data Carpentry, or Library Ca
 *   it covers the core topics (see below),
 *   at least one of the instructors is a certified Carpentries Instructor,
 *   you run our standardized pre- and post-workshop assessments and ensure everyone participates, and
```
Comment (alex-konovalov) [inline _episodes/21-carpentries.md:108]: This sentence is now incomplete.

Comment (brownsarahm) [reply _episodes/21-carpentries.md:108]:
```suggestion
*   you run our standardized pre- and post-workshop assessments and ensure everyone participates.
```

**Comment by sheraaronhurt** [comment]
@brownsarahm thanks for noting that in the instructor checklist. I will get that removed from the handbook.

**Comment by karenword** [comment]
I think in this particular case, since this is about branding, we should go ahead and remove the language regarding attendance. @sheraaronhurt I think you might consider leaving the checklist in the handbook as-is, since that specifically refers to supporting the host and does not suggest that attendance records are required by us.

**Pull request merged**